javascript/AutocompleteField.js

```diff
@@ -3,46 +3,79 @@
  * This is not strictly using jQuery, like the rest of the CMS.
  */
 
-(function($) {
-	$(function() {
+(function($) {
+	$.entwine('ss.autocomplete', function($){
+
+		$('.field.autocomplete input.text').entwine({
+
+    		onmatch: function() {
 
-		// Load autocomplete functionality when field gets focused
-		$('.field.autocomplete input.text').live('focus', function() {

```

Comment:
If using entwine you'll need to include the necessary libs in the requirements call, please.